INTHEWILD.md

```diff
@@ -141,6 +141,7 @@ Currently, **officially** using Airflow:
 1. [Coursera](https://coursera.org/) [[@darrenmk](https://github.com/darrenmk)]
 1. [CoverGenius](https://covergenius.com) [[@lihan](https://github.com/lihan), [@tyomo4ka](https://github.com/tyomo4ka), [@girish-l](https://github.com/girish-l)]
 1. [Craig@Work](https://www.craigatwork.com)
+1. [CrazyLabs](https://www.crazylabs.com/) [[@hadars](https://github.com/HadarSha), [@gilt](https://github.com/gil-tober)]
```

Comment:
```suggestion
1. [CrazyLabs](https://www.crazylabs.com/) [[@HadarSha](https://github.com/HadarSha), [@gil-tober](https://github.com/gil-tober)]
```
Looking at other records, it looks like these are consistent with the GitHub handle included in the URL